examples/proc_macro.rs

```diff
@@ -64,18 +71,17 @@ async fn main() -> anyhow::Result<()> {
 	let url = format!("ws://{}", server_addr);
 
 	let client = WsClientBuilder::default().build(&url).await?;
-	assert_eq!(client.storage_pairs(10, None).await.unwrap(), vec![1, 2, 3, 4]);
+	assert_eq!(client.storage_keys(vec![1, 2, 3, 4], None::<ExampleHash>).await.unwrap(), vec![vec![1, 2, 3, 4]]);
 
-	let mut sub = client.subscribe_storage(None).await.unwrap();
-	assert_eq!(Some(vec![]), sub.next().await.unwrap());
+	/*let mut sub: Subscription<Vec<ExampleHash>> =
+		RpcClient::<ExampleStorageKey, _>::subscribe_storage(&client, None).await.unwrap();
+	assert_eq!(Some(vec![[0; 32]]), sub.next().await.unwrap());*/
```

Comment:
work-in-progress I assume?

Reply:
yes, the entire example doesn't work.

I have some issues with adding trait bounds to `subscription output` i.e, `item = SomeType` which can be contain generics.